net.runelite.RuneLite.appdata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Also, please add [OARS](https://hughsie.github.io/oars/) metadata to the appdata file.